ftw/theming/configure.zcml

```diff
@@ -26,6 +31,19 @@
         provides="Products.GenericSetup.interfaces.EXTENSION"
         />
 
+    <genericsetup:registerProfile
+        name="uninstall"
+        title="ftw.theming"
```

Comment:
Please change to `ftw.theming (uninstall)` for beeing clear which profile it is 😉